### PR TITLE
Change Service Endpoints

### PR DESCRIPTION
**Pull request by lomaxap**
- removes `services` from endpoints
- pass `ids` param as comma-separated string
- check if missing_veterans exist

### CODE REVIEW

File: app/services/external_api/va_dot_gov_service.rb [hunk 1]
```diff
@@ -72,11 +72,11 @@ def base_url
     end
 
     def facilities_endpoint
-      "services/va_facilities/v0/facilities"
+      "va_facilities/v0/facilities"
```
Comment (mdbenjam) [inline app/services/external_api/va_dot_gov_service.rb]: This needs to use the ENV var. It should be available now.

File: app/jobs/fetch_hearing_locations_for_veterans_job.rb [hunk 2]
```diff
@@ -68,6 +68,17 @@ def perform
       rescue Caseflow::Error::VaDotGovLimitError
         sleep 60
         va_dot_gov_address = validate_veteran_address(veteran)
+      rescue Caseflow::Error::VaDotGovRequestError => e
+        case e.message["messages"][0]["key"]
+        when "AddressCouldNotBeFound"
+          # do something when address is not valid (1234 Fake st.)
```
Comment (mdbenjam) [inline app/jobs/fetch_hearing_locations_for_veterans_job.rb]: At the very least we should sent the exception to sentry: `Raven.capture_exception(e)`, but I think failing the job is fine. It'll force us to work through the various errors.

File: app/services/external_api/va_dot_gov_service.rb (hunk 1 above)
Comment (lomaxap) [reply app/services/external_api/va_dot_gov_service.rb]: i don't see the endpoints there, just the base url

File: app/jobs/fetch_hearing_locations_for_veterans_job.rb [hunk 3]
```diff
@@ -19,8 +19,10 @@ def file_numbers
   end
 
   def missing_veteran_file_numbers
-    existing_veteran_file_numbers = Veteran.where(file_number: file_numbers).pluck(:file_number)
-    (file_numbers - existing_veteran_file_numbers)[0, (QUERY_LIMIT - existing_veteran_file_numbers.length)]
+    @missing_veteran_file_numbers ||= begin
```
Comment (mdbenjam) [inline app/jobs/fetch_hearing_locations_for_veterans_job.rb]: what's the begin for?

File: app/services/external_api/va_dot_gov_service.rb (hunk 1 above)
Comment (mdbenjam) [reply app/services/external_api/va_dot_gov_service.rb]: 🤦‍♂️ my bad, thought this was the base url stuff. Totally right. Didn't see the function above.

**Pull request merged**